Translator Script/Translation.py

```diff
@@ -0,0 +1,13 @@
+# First Step : Go to cmd and type: pip install googletrans==3.1.0a0

```

Comment:
This first line is not needed as it is already mentioned in the README.md file dependency section